src/components/order-service/order-service.controller.ts

```diff
@@ -1,31 +1,40 @@
 import * as angular from 'angular';
+import * as _ from 'lodash';
 
 export class OrderServiceController implements angular.IController {
 
-  static $inject = ['$scope'];
+  static $inject = ['$scope', '$filter', 'DataService', 'Logger'];
 
   public ctrl: any = this;
   public $scope: any;
 
-  constructor($scope: any) {
+  private $filter: any;
+  private DataService: any;
+  private Logger: any;
+
+  constructor($scope: any, $filter: any, DataService: any, Logger: any) {
     this.$scope = $scope;
+    this.$filter = $filter;
+    this.DataService = DataService;
+    this.Logger = Logger;
   }
 
   public $onInit() {
-    this.ctrl.serviceIcon = this.ctrl.service.icon ? this.ctrl.service.icon : 'pf pficon-service';
-    this.ctrl.serviceName = this.ctrl.service.name;
+    this.ctrl.serviceIcon = this.ctrl.serviceClass.icon || 'fa fa-cubes';
```

Comment:
I believe these fields are now called:  'iconClass' and 'imageUrl', where 'imageUrl' should be used if defined.